### PR TITLE
Awesome books classes refactor

### PR DESCRIPTION
**Pull request by Christelle-12**
-I and my coding partner created a branch "awesome-books-classes" for this project.
-I and my coding partner refactor the javascript code and used classes.
-I and my partner created contractors for the classes.
-I and my coding partner used class methods to add and remove books.
-I and my coding partner stylized the wireframe page.
-I and my coding partner checked for linter errors.

### CODE REVIEW

**Review by Shisui6 — COMMENTED** [review]
Hi @Christelle-12 and @BranBayou, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛠️

You have submitted a project, that:
- Does not have a professional PR description. Please update your PR description to include the details of all the work done in this branch
- Does not have a professional readme file. The information in your readme file is not for this project and is unrelated. Please update your readme file to include information about the awesome books project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: scripts.js [hunk 1]
```diff
@@ -1,61 +1,74 @@
 /* eslint-disable no-use-before-define */
```
Comment (Wes-Isaac) [inline scripts.js]: - [ ] Great job so far, kindly don't disable linters, if issues come up let's try to fix them.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @Christelle-12 & @BranBayou, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Great communication with the reviewer 💯.
- Professionally documented your work 🥇.
- LInters setup properly ✅.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by topeogunleye — APPROVED** [review]
### Hi @BranBayou, @Christelle-12  

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Removed disabling of linters :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**